packages/frontend/index.d.ts

```diff
@@ -161,5 +161,6 @@ declare namespace JSX {
         'amp-soundcloud': any;
         'amp-iframe': any;
         template: any;
+        'amp-fallback': React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement> & { fallback?: string };
```

Comment:
Interesting! So my immediate thougthts are. 

I like:

* no longer globally modifying basic html types
* more discoverable/explicit

The cons:

* it is quite brittle (must be a div, cannot compose easily)

So I'm a bit torn. In a a way I like the AMPAttributes approach because it allows us to be flexible in the html we write (i.e. write it like we'd want to).

To support the breadth of AMP behaviour here, we'd have to create aliases for quite a few html tags potentially?

Reply:
I don't think this tag would behave as a div as it's unrecognised by the browser so might exhibit more span-like behaviour.